app/views/va_facilities/_facility_results_table_header.html.erb

```diff
@@ -0,0 +1,84 @@
+  <th scope="col"><a class="fake-link" id="toggle_by_facility">Facility
+    <% if params[:asc].blank? || params[:sortby] == "facility" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
+        &#9650;
+      <% end %>
+    <% end %>
+  </a>
+  </th>
+  <th scope="col"><a class="fake-link" id="toggle_by_common_name">Common name
+    <% if params[:sortby] == "common_name" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
+        &#9650;
+      <% end %>
+    <% end %>
+  </a>
+  </th>
+  <th scope="col"><a class="fake-link" id="toggle_by_visn">VISN
+    <% if params[:sortby] == "visn" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
+        &#9650;
+      <% end %>
+    <% end %>
+  </a>
+  </th>
+  <th scope="col"><a class="fake-link" id="toggle_by_type">Type
+    <% if params[:sortby] == "type" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
+        &#9650;
+      <% end %>
+    <% end %>
+  </a>
+  </th>
+  <th scope="col"><a class="fake-link" id="toggle_by_practices_created">Created
+    <% if params[:sortby] == "practices_created" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
+        &#9650;
+      <% end %>
+    <% end %>
+  </a>
+  </th>
+  <th scope="col"><a class="fake-link" id="toggle_by_adoptions">Adopted
+    <% if params[:sortby] == "adoptions" %>
+      <% if params[:asc] == "true" %>
+        &#9650;
+      <% end %>
+      <% if params[:asc] == "false" %>
+        &#x25BC;
+      <% end %>
+      <% if params[:asc].blank? %>
+        &#9650;
+      <% end %>
+    <% end %>
+  </a>
+  </th>
```

Comment:
Can we make the `<th></th>` a partial and then use local variables to change the column name, `sortby` option, and `id`?